src/resolver.ts

```diff
@@ -33,3 +33,7 @@ export const hashFile = (url: string) => {
   );
   return hashHex;
 };
+
+export function ensureNoTrailingSlash(url: string) {
```

Comment:
Maybe name this `stripTrailingSlash` as I feel `ensure` implies that the function throws.

Reply:
Done